XamlControlsGallery/ControlPages/NavigationViewPage.xaml

```diff
@@ -74,6 +75,10 @@
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="Auto"/>
             <RowDefinition Height="Auto"/>
+            <RowDefinition Height="Auto"/>
```

Comment:
I wonder if this whole thing could be replaced with a stack panel?

Reply:
Maybe? I had assumed Yulia made it a Grid for a reason so I didn't want to change it. But I can look into it.

Reply:
Done- changed to StackPanel in upcoming commit.